style.css

```diff
@@ -0,0 +1,733 @@
+    row-gap: 50px;
+    margin-top: 5em;
+    margin-bottom: 5em;
+  }
+
+  .work .card {
+    width: 90%;
+    height: 95%;
+    border-radius: 20px;
+    margin-bottom: 2em;
+    margin-left: 5%;
+  }
+
+  .work .card .card-container {
+    width: 96%;
+    height: 98%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-left: 3%;
+  }
+
+  .card .card-container img {
+    width: 48%;
+    height: 90%;
+  }
+
+  .card-container .card-items {
+    width: 50%;
+    height: 92%;
+    align-items: center;
+  }
+
+  .card-items h2 {
+    font-size: 40px;
+    width: 100%;
+    margin: 0.5em 0;
+  }
+
+  .card-items .work-details {
+    width: 60%;
+    min-width: 300px;
+    margin-bottom: 2em;
+  }
+
+  .card-items .language {
+    margin: 1em 0;
+  }
+
+  .card1 .language,
+  .card2 .language {
+    width: 90%;
+    min-width: 350px;
+    justify-content: start;
+    flex-wrap: wrap;
+  }
+
+  .card-items button {
+    background-color: #4053fc;
+    color: #fff;
+  }
+
+  .card-items button:hover {
+    background-color: darkblue;
+    transition: all 0.5s;
+  }
+
+  .card3 {
+    grid-area: 1/1/2/2;
+  }
+
+  .card4 {
+    grid-area: 2/1/3/2;
+  }
+
+  .card1 {
+    grid-area: 3/1/4/2;
+  }
+
+  .card2 {
+    grid-area: 4/1/5/2;
+  }
+
+  .card2 .card-container,
+  .card4 .card-container {
+    flex-direction: row-reverse;
+  }
+}
+
+/* This is about myself section  */
+
+.about-myself {
+  font-family: 'Poppins', sans-serif;
+  height: auto;
+  background-color: #ffff;
+  border-top-right-radius: 80px;
+}
+
+.about-myself .contact button {
+  font-size: 18px;
+  font-weight: 500;
+  width: 162px;
+  height: 48px;
+  color: #396df2;
+  background-color: #fff;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  padding: 12px;
+  margin: 20px 0;
+  cursor: pointer;
+}
+
+.about-myself .contact button:hover {
+  background-color: #396df2;
+  color: #fff;
+  transition: all 0.5s;
+}
+
+.details {
+  padding-bottom: 30px;
+}
+
+.details .div-container {
+  padding-bottom: 10px;
+  margin: 1em auto;
+}
+
+.frameworks-div .frameworks,
+.skills-div .skills {
+  display: none;
+}
+
+.details .details-header {
+  width: 90%;
+  height: 50px;
+  line-height: 50px;
+  display: flex;
+  justify-content: space-between;
+  padding: 1em auto;
+}
+
+.div-container.hide {
+  border-bottom: 1px solid #999;
+}
+
+.details-header h2,
+.details-header span {
+  cursor: pointer;
+}
+
+.right {
+  transform: rotate(-90deg);
+}
+
+.details ul {
+  list-style-type: none;
+}
+
+.details ul li {
+  font-size: 18px;
+  line-height: 20px;
+  width: 100%;
+  height: 67px;
+  color: #253858;
+  background-color: #f7f7f9;
+  margin: 15px auto;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+}
+
+.details ul li img {
+  width: 50px;
+  height: 50px;
+  background-color: lightgreen;
+  border-radius: 50%;
+  margin: auto  1em;
+}
+
+/* for smaller screen view  */
+@media screen and (max-width: 375px) {
```

Comment:
[Optional] You can remove this max-width @media query rule since we only need two @media rules for this project. The recommended approach would be to build the mobile version first and add styles for the mobile version with min-width: 375, and after that, for the desktop version, you can update your styles with min-width: 768px. All the styles you add in the mobile version will automatically apply in the desktop version. So, if there's something you need to update in the desktop version, you can update that specific style. 😁 👍